CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#1841](https://github.com/nix-rust/nix/pull/1841))
 - Added `eaccess()` on FreeBSD, DragonFly and Linux (glibc and musl).
   ([#1842](https://github.com/nix-rust/nix/pull/1842))
+- Added `SockProtocol::Raw = libc::IPPROTO_RAW` for raw sockets
```

Comment:
```suggestion
- Added `SockProtocol::Raw` for raw sockets
```

Reply:
@asomers I did some errors with rebase, but now looks likes CI is fine. Also merge conflict is now resolved